packages/cmf/src/middlewares/http/middleware.js

```diff
@@ -160,6 +168,15 @@ export const httpMiddleware = ({ dispatch }) => next => action => {
 						errorObject.stack.response = response;
 						errorObject.stack.messageObject = JSON.parse(response);
 					} finally {
+						switch (errorObject.stack.status) {
+							case HTTP_STATUS.UNAUTHORIZED:
+							case HTTP_STATUS.FORBIDDEN:
+							case HTTP_STATUS.NOT_FOUND:
+								dispatch(on40xError(errorObject.stack.status));
```

Comment:
Should we dispatch 40x Error now when we have a onError function passed to the httpAction ? 
( @jmfrancois )